switches/http/http.cpp

```diff
@@ -0,0 +1,1280 @@
+
+/*
+ asebahttp - a switch to bridge HTTP to Aseba
+ 2014-12-01 David James Sherman <david dot sherman at inria dot fr>
+ 
+ Provide a simple REST interface with introspection for Aseba devices.
+ 
+    GET  /nodes                                 - JSON list of all known nodes
+    GET  /nodes/:NODENAME                       - JSON attributes for :NODENAME
```

Comment:
It is not clear to me whether `:NODENAME` means variable `NODENAME` or variable `:NODENAME`, and whether `:` should be put into the URL.